pkg/insights/resyncer.go

```diff
@@ -222,6 +222,10 @@ func (r *resyncer) createOrUpdateServiceInsight(mesh string) error {
 			// Mesh no longer exist so we cannot upsert the insight for it.
 			return nil
 		}
+		if store.IsResourceConflict(err) {
+			log.V(1).Info("ServiceInsight resource conflict")
```

Comment:
again, sounds scary. I think we should say that it will retry soon

Reply:
Fixed, thanks!